web/src/components/Education/Education.jsx

```diff
@@ -22,6 +22,25 @@ export const Education = ({ data, onEducationChange }) => {
     return newErrors;
   };
 
+  const formatToMonthYear = (val) => {
+    const [year, month] = val.split('-');
+    const months = [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ];
+    return months[parseInt(month, 10) - 1] + ' ' + year;
+  };
+
```

Comment:
This is obsolete code and should be removed as this function is now in https://github.com/SyedArslanHaider/smart-cv-builder/blob/main/web/utils/date.js